docs/DECISION-RECORD.md

```diff
@@ -63,6 +63,18 @@ In onderstaande afbeelding is gedocumenteerd welke gegevens uit de verschillen Z
 
 ![image](https://raw.githubusercontent.com/Klantinteractie-Servicesysteem/.github/add-decision-record/docs/images/MappingZaakDetail.png)
 
- ## Logging voor verwerking
+## Logging voor verwerking
 Omdat er nog geen definitieve standaard voor logging voor verwerking was in KISS Fase 1c, is er een voorbereiding gedaan t.b.v. een API voor verwerkingslogging, om de algemene werking alvast op te zetten totdat er een standaard beschikbaar is. Elke keer dat er een api call wordt gedaan naar externe systemen loggen we UserID, datum/tijd, het api-endpoint dat werd aangeroepen, en de methode waarmee het endpoint werd aangeroepen. Deze logging staat los van de errorlogging. Er is een zeer basaal API-endpoint voor ingericht. 
 
+## Autorisatie
```

Comment:
wat er staat klopt, maar ik mis hier een beetje de decision.
wat was hier nou eigenlijk ook al weer het probleem? was het dat de esuite niet op applicatie niveau, maar alleen op userniveau kan autoriseren? waarom was dat een probleem voor en wat hebben we gedaan om dat op te lossen., ik meen dat we alleen de optie hebben toegevoegd om bij overigeobjecten api calls een jwt token ipv een apikey mee te sturen.

Reply:
Ik heb een aanapssing gedaan en geprobeerd iets duidelijker de aanleiding te omschrijven. De vraag "wat hebben we gedaan om dat op te lossen." vind ik lastig goed te formuleren, omdat ik zelf onvoldoende begrijp wat het verschil is tussen Objecten API enerzijds en Klanten/Zaken anderzijds. 

Kun jij dat beter omschrijven?